examples/log-attack1/index.sol

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: MIT
+pragma abicoder v2;
+
+pragma solidity ^0.8.0;
+
+contract Inc {
+    uint256 public count = 0;
+
+    event Increment(address who);   
+
+    function increment() public returns (uint256) {
+        emit Increment(msg.sender);
```

Comment:
Your goal is to emit an event identical to the one that the Reach compiler produces. Look at the generated solidity (using build intermediates) to see what that is

Reply:
That’s what am doing now

Reply:
No, because Reach does not generate an event named `Increment`

Reply:
I meant that’s what **I’m working on right now

Reply:
Ooooooo, I thought you meant "now" = "what this commit is". Sorry :P